doc/compiler/regular_expression/Regular_Expression.md

```diff
@@ -0,0 +1,41 @@
+# Regular expression
+
+A **regular expression** or **regex** is a concise way to describe matching or searching criteria for text strings. For more on regular expressions in general, see the [Wikipedia article](https://en.wikipedia.org/wiki/Regular_expression).
+
+Testarossa's `-Xjit` option uses regular expressions for many different debugging facilities. It should be noted that Testarossa implements a simplified regex matcher and it works a little differently compared to complete regex matchers. These regular expressions are not the same as those used by Perl or grep.
```

Comment:
The Eclipse OMR compiler `-Xjit` option allows a simplified form of regular expressions as arguments to some options.  The regular expressions are not the same as those used by Perl or grep but behave more like shell globs.